releases/release-1.18/README.md

```diff
@@ -1,3 +1,9 @@
+---
+title: "Kubernetes 1.18 Release"
+weight: 98
+slug: "kubernetes-118-release" 
```

Comment:
```suggestion
slug: "release" 
```

The slug for this should just be `release` or `current-release`. The filename will be changed during the import to match.

Reply:
Making the change 👍

Reply:
Done !